src/ui/public/timepicker/timepicker.html

```diff
@@ -40,58 +40,89 @@
                 <div class="kbn-timepicker-section">
                   <label>
                     From:
-                    <span ng-show="relative.preview">{{relative.preview}}</span>
-                    <span ng-hide="relative.preview"><i>Invalid Expression</i></span>
+                    <span ng-show="relative.from.preview">{{relative.from.preview}}</span>
+                    <a class="label label-default" ng-click="setRelativeToNow('from')">Set To Now</a>
```

Comment:
As it currently stands, it doesn't make much sense to add the "Set to Now" button in the "From" field, because there's no way to have a "To" that's in the future (as far as the relative picker is concerned). However, since we [just added it to the absolute timepicker](https://github.com/elastic/kibana/pull/10985), and since there's an [open issue to add future relative dates](https://github.com/elastic/kibana/issues/6732), I'm fine with it.